docs/business/alternate_domains.rst

```diff
@@ -0,0 +1,32 @@
+Alternate Domains
+=================
+
+Read the Docs supports a number of custom domains for your convenience.
```

Comment:
This can probably just be removed. It's sort of a weird sentence, as there aren't multiple types of custom domains, and the sentence doesn't really add anything of value to the information in this doc.